### PR TITLE
create Rot13.go

### PR DESCRIPTION
**Pull request by aakashdinkar**

### CODE REVIEW

File: ciphers/Rot13.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package main
+
+import "fmt"
+       
+
+func dencrypt(str string, key int)string{
```
Comment (brayo-pip) [inline ciphers/Rot13.go:6]: fix typos
```suggestion
func denrypt(str string, key int)string{
```

File: ciphers/Rot13.go [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    	  }
+	}
+	return char
+}
+
+func main(){
+	fmt.Print("Enter your Message:")
+	var str string
+	fmt.Scan(&str)
+	str_d := dencrypt(str,13)
```
Comment (brayo-pip) [inline ciphers/Rot13.go:25]:
```suggestion
	str_d := decrypt(str,13)
```

File: ciphers/Rot13.go [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+	fmt.Println("Decryption using key => 13 is :",str_d)
+	str_e := dencrypt(str_d,13)
```
Comment (brayo-pip) [inline ciphers/Rot13.go:27]:
```suggestion
	str_e := decrypt(str_d,13)
```

**Comment by tjgurwara99** [comment]
Please take a look at our new [contribution guidelines](https://github.com/TheAlgorithms/Go/blob/d2f12849df76203d80762ceca5105531d3424384/CONTRIBUTING.md) and make changes accordingly. Currently this PR is not up to standard. If no changes are made in the next two weeks, I will consider this PR to be stale and will close this PR then.

**Pull request closed without merging**